src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,42 +29,43 @@ export function getStaticProps(context) {
 
-Because Amplify builds on the AWS CDK, any of the 175+ AWS services can be added to your app using [custom resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) and L2/L1 AWS CDK constructs.
+**Should I use Amplify Gen 1 or Gen 2 in new apps?**
 
-**We are planning to build and release a new application before the code-first developer experience is generally available. Which version of Amplify should we use?**
+If you're building a new app, we recommend you use Amplify Gen 2.
 
-We recommend using Amplify’s tooling-first (Gen 1) developer experience with the CLI and Studio to get access to our complete feature set. If your primary use cases are configuring Auth or Data, we recommend starting with our [L3 CDK constructs](/[platform]/reference/cdk-constructs/) that power the Gen 2 experience.
+**Does Amplify Gen 2 support DataStore?**
 
-**Is Gen 2 backward compatible with existing Amplify projects on Gen 1?**
+Amplify code-first DX (Gen 2) supports GraphQL APIs without DataStore. We will release migration support for moving DataStore Gen 1 apps to Gen 2.
 
-No. Gen 2 apps adopt a file-based convention with a much smaller footprint. You cannot use the Gen 1 CLI to update the `resource.ts` files in Gen 2.
+**What programming languages does Amplify Gen 2 support?**
 
-**Is there a way to upgrade an existing Amplify project from Gen 1 to Gen 2?**
+Amplify Gen 2 supports a wide range of programming languages for client-side development. This includes dedicated client-side libraries for JavaScript, TypeScript, Dart, Java, Kotlin, and Swift. For backend development, Amplify Gen 2 uses TypeScript.
+
+**In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-While Gen 2 is in public preview, we will provide migration guides that describe how to manually move from Gen 1 to Gen 2. When generally available, we will offer tooling that will help developers move projects from Gen 1 to Gen 2.
+Amplify code-first DX (Gen 2) supports Auth, Data, Storage, and Functions. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs which there is documentation for under the respective category name.
 
-**What platforms will Gen 2 support?** 
+**What if we want to add a feature like AI/ML or Amazon Location Service to our application in Gen 2?**
 
-While in public preview, Gen 2 supports a TypeScript-first developer experience for building backends. Teams that have built backends with Gen 2 can reuse that backend in mobile apps if desired.
+Because Amplify builds on the AWS CDK, any of the 175+ AWS services can be added to your app using [custom resources](/[platform]/build-a-backend/add-aws-services/custom-resources/) and L2/L1 AWS CDK constructs.
```

Comment:
I don't fully get this question. I think it's answered by the previous question.

Reply:
If you're reading FAQs top to bottom yes, but I think these are two different topics 1/ what happens to the non data/auth/functions/storage categories? and is extensibility possible? I think most FAQs have questions with similar answers but different questions. Plus if someone isn't familiar with CDK they may not know this.